wordle-web/components/Keyboard.vue

```diff
@@ -7,9 +7,12 @@
   >
     <v-col
       cols="auto"
+      md="auto"
+      sm="auto"
+      xs="auto"
```

Comment:
The use of auto for various sizes doesn't actually get you anything. The grid system will default to cols="auto", but also gets set conditionally to "auto" depending on the screen size. That is, you have conditional logic for an assignment that can only ever be "auto" via the current logic.

It would've been more effective to either control (via columns) or directly change the rendering size of the keys dependent on the screen size (md, sm xs). I would reckon that changing the size of the keys directly would be easier with this setup.

I believe it's the static key size that is making it appear to be responsive when you swap to a new device without reloading the page. When you select a new rendering device, the keys are shrunk proportionally to the new screen size and they look good, but when you re-render the page the keys are rendered too large to gracefully form the expected rows with the auto-width columns

(Confirmed on my personal Samsung Super Deluxe Mega Galaxy Chill S22 Ultra)